### PR TITLE
feat: move credential delete forever button to inactive tab

### PR DESCRIPTION
**Pull request by agatha197**
### TL;DR
The delete button that was visible in the active tab of credentials has been moved to the inactive tab. This change is to align with the vfolder list and improve usability.

### Screenshots
| Before | After |
|--------|--------|
| image | image | 
| image | image |

### How to test
1. Log in with the superadmin user.
2. Navigate to the user's credentials tab.
3. Check the control column on both the active and inactive tabs.

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [x] Specific setting for review (eg., KB link, endpoint or how to setup) 
- [x] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by agatha197** [comment]
* **#2577** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2577?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @agatha197 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by ironAiken2 — APPROVED** [review]
LGTM!
